### DISCUSSION TITLE
Vendor

### DISCUSSION THREAD

**Discussion by BreadDain** [IHateMyKite/UnforgivingDevices#15] [Ideas]
Decided to make a small experiment to see how it goes, also get myself a little knowledge. Added abadon vendor to the eastern exit from Whiterun, near 2 bridges. He will always have a black face unless his facegen is generated, easy to do, didn't do that just to keep it a single file. Sells mod restraints and reagents + buys any devices, should be immune and peaceful to anything.

Couldn't upload it here so here is Discord link. If you think it is worth you can incorporate it, otherwise just ignore. I might try to make a separate shop for it, just as practice.
https://cdn.discordapp.com/attachments/482228296617295882/965659304529117274/UnforgivingDevices_-_Vendor.esp

![изображение](https://user-images.githubusercontent.com/30041703/163844303-038a8777-f25c-41ab-8e1a-2739ba37ec82.png)

**Comment by IHateMyKite** [comment]
I don't think mod needs vendor right now, but it will surely need more NPCs in future. What did you use as reference for creating the NPC?

**Comment by BreadDain** [comment]
Used this video https://youtu.be/EUcb7SJuctY
He has a plenty of instructions on AI packages and NPC interactions
